api/v1/controllers/CachingController.js

```diff
@@ -0,0 +1,184 @@
+//This Controller is used for testing purposes
```

Comment:
Let's remove this if testing is complete. We'll have unit tests to permit more conventional testing soon.